modules/core/core-scheduler/src/main/java/com/enonic/xp/impl/scheduler/distributed/SchedulableTask.java

```diff
@@ -48,6 +57,18 @@ public ScheduledJob getJob()
         return job;
     }
 
+    private static Context adminContext()
```

Comment:
Why static?

Reply:
To keep no reference to instance

Reply:
Method is private

Reply:
also shows adminContext result don't need any information from particular instance

Reply:
We have a few duplicates of such method. We would need to move it to utility class one day, anyway.